src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -69,6 +69,7 @@ def load_arguments(self, _):
         c.argument('name', options_list=['--name', '-n'], help="Name of the new app service plan", completer=None)
         c.argument('sku', arg_type=sku_arg_type)
         c.argument('is_linux', action='store_true', required=False, help='host webapp on Linux worker')
+        c.argument('hyper_v', action='store_true', required=False, help='Preview - host webapp on Windows container')
```

Comment:
Will a argument / parameter name of is_windows be clearer than hyper_v? Did you test help to see what the parameter shows up as?

I am assuming `hyper_v` is a counterpart to `is_linux`, right?

Reply:
`hyper-v` is indeed `is_linux` counterpart. `is_windows` will be confusing since non-container web apps are Windows based. We chose a term that differentiates from Linux web apps and from Windows "classic" apps. By the way, we are using the same term in the equivalent PowerShell cmdlet for consistency.

This is what help loos like: 
`--hyper-v                      : Preview - host webapp on Windows container.`

Reply:
I see, thanks for the clarification.